DEVELOPER_GUIDE.md

```diff
@@ -64,13 +64,13 @@ If it's the only version of node installed, it will automatically be set to the
 
 #### Install `yarn`
 
-To install yarn run:
+OpenSearch Dashboards is set up using yarn v1. To install it, run:
 
 ```bash
-$ npm i -g yarn
+$ corepack enable
```

Comment:
Previously, we had `corepack prepare ...`. The `prepare` subcommand doesn't exist in newer versions of corepack: https://github.com/nodejs/corepack?tab=readme-ov-file#utility-commands

Instead, the `enable` subcommand should add all relevant package managers (including yarn).